content/blog/play-the-long-game/index.md

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
Changes in this file were to separate out the focus of this post (long-term thinking vs. daily grind)